manifests/ns-reconciler-cluster-role.yaml

```diff
@@ -26,6 +26,9 @@ rules:
 - apiGroups: ["configsync.gke.io"]
   resources: ["reposyncs/status"]
   verbs: ["get","list","watch","update","patch"]
+- apiGroups: [""]
+  resources: ["configmaps"]
+  verbs: ["get","list","watch","create","update","patch"]
```

Comment:
Why does ns-reconciler need create/update/patch?

Reply:
In order to mount the ConfigMaps (which are in the reposync's namespace) to the pod (which is in the config-management-namespace), it needs to copy the ConfigMap over.

Reply:
Doesn't the reconciler-manager upsert the ConfigMap to c-m-s?

Reply:
Actually, with the approach of mounting the ConfigMaps into the Deployment - I don't think the ns-reconciler should even need get/list/watch

Reply:
Ah that's right, I think I confused the roles of the ns-reconciler and the reposync controller which is part of the reconciler-manager. I think we can take out the extra permissions I added, I will double check with some testing tomorrow

Reply:
Did some manual testing and yup, these permissions aren't necessary. Took them out.